pages/workspace/workspace.ejs

```diff
@@ -28,32 +28,29 @@
 
     <div class="collapse navbar-collapse" id="workspace-nav">
       <ul class="navbar-nav ml-auto">
-        <!--
-        <li class="nav-item ml-2">
-          <a id="reload" class="nav-item btn btn-light" href="#">
-            <i class="fas fa-redo text-primary" aria-hidden="true"></i>
-            Reload
-          </a>
-        </li>
-        -->
-        <li class="nav-item ml-2">
-          <!-- this looks horrible -->
-          You can refresh this tab at any time, or close and re-open it.
-        </li>
-        <li class="nav-item ml-2">
+        <li class="nav-item ml-2 my-1">
           <a id="reboot" class="nav-item btn btn-light" href="/workspace/<%= workspace_id %>/reboot">
             <i class="fas fa-sync text-info" aria-hidden="true"></i>
             Reboot
           </a>
         </li>
-        <!--
-        <li class="nav-item ml-2">
-          <a id="close" class="nav-item btn btn-light" href="#">
-            <i class="fas fa-power-off text-danger" aria-hidden="true"></i>
-            Close
-          </a>
+        <li class="nav-item ml-2 my-1">
+          <button type="button" class="nav-item btn btn-light" data-container="body" data-toggle="popover" data-placement="left" data-html="true"
+              data-content='<p class="p-1 m-0"><span class="badge badge-secondary mr-1">Question <%= question_id %></span><%= question_title %></p>'>
```

Comment:
We don't expose the `question_id` to the user because it's not really informative. For instructors we normally list the `qid` and for students we list the question number within the context of the assessment. I suggest for this PR that we simple strip out this information button and leave that for a future PR. It's not essential for this MVP.

Reply:
Done -- reverted to previous but just with the Help button in the corner.